### PR TITLE
[Enhancement] - Specify Set Value for Different Keys

### PR DESCRIPTION
**Pull request by FaaizHaikal**
## Description

Each key has its own preferential value change upon button click.

## Type of Change

- [ ] Bugfix
- [x] Enhancement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause the existing functionality to not work as expected)

## How Has This Been Tested?

- [ ] New unit tests added.
- [x] Manual tested.

## Checklist

- [x] Using Branch Name Convention
  - `feature/JIRA-ID-SHORT-DESCRIPTION` if has a JIRA ticket
  - `enhancement/SHORT-DESCRIPTION` if has/has no JIRA ticket and contain enhancement
  - `hotfix/SHORT-DESCRIPTION` if the change doesn't need to be tested (urgent)
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have made the documentation for the corresponding changes.

### CODE REVIEW

File: src/components/NumberField.jsx [hunk 1]
```diff
@@ -37,12 +37,54 @@ function NumberField(props) {
...
-      setWalkingValue(name, keys, value + val);
+      switch (name) {
+        case 'balance':
+          value_change /= 10;
+          break;
+        case 'pid':
+          value_change *= 10;
+          break;
+        default:
+          break;
```
Comment (threeal) [inline src/components/NumberField.jsx]: Unnecessary lines.
```suggestion
```

File: src/components/NumberField.jsx [hunk 2]
```diff
@@ -37,12 +37,54 @@ function NumberField(props) {
...
+      switch (name) {
+        case 'offset':
+          switch (keys) {
+            case 'x_offset':
+            case 'y_offset':
+            case 'z_offset':
+              value_change *= 10;
+              break;
+            default:
+              break;
```
Comment (threeal) [inline src/components/NumberField.jsx]: Unneccessary lines:
```suggestion
```

File: src/components/NumberField.jsx [hunk 3]
```diff
@@ -37,12 +37,54 @@ function NumberField(props) {
...
+              value_change *= 10;
+              break;
+            default:
+              value_change /= 10;
+              break;
+          }
+          break;
+
+        default:
+          break;
```
Comment (threeal) [inline src/components/NumberField.jsx]: Unnecessary lines:
```suggestion
```

File: .gitignore [hunk 4]
```diff
@@ -23,3 +23,4 @@ dist-ssr
 *.sln
 *.sw?
 envoy.yaml
+.env
```
Comment (threeal) [inline .gitignore]: I suggest ignoring all dot-prefixed files except for important ones (e.g., this `.gitignore` file itself):
```suggestion
.*
!.gitignore
```

Other dot-prefixed files like `.vscode/*`, `.idea`, etc., may no longer need to be specified.

**Review by JayantiTA — APPROVED** [review]

**Pull request merged**